stackslib/src/config/mod.rs

```diff
@@ -4152,7 +4350,15 @@ pub struct InitialBalance {
 #[derive(Clone, Deserialize, Default, Debug)]
 #[serde(deny_unknown_fields)]
 pub struct InitialBalanceFile {
+    /// The Stacks address to receive the initial STX balance.
+    /// Must be a valid Stacks address string (e.g., "ST2QKZ4FKHAH1NQKYKYAYZPY440FEPK7GZ1R5HBP2").
```

Comment:
I'm tempted to suggest `.. a valid Stacks testnet address` but that may not be clear to an end user/dev if they're using mocknet etc. 
would `... a valid non-mainnet Stacks address` work here possibly?